contentcuration/contentcuration/static/js/edit_channel/import/views.js

```diff
@@ -128,10 +130,14 @@ var ImportModalView = BaseViews.BaseView.extend({
 
     _startImport: function() {
         var self = this;
-        function onFinishImport(resolve) {
-            self.once('finish_import', function() {
+        function onFinishImport(resolve, reject) {
+            self.once('finish_import', function(failed = false) {
```

Comment:
Not sure if this syntax is allowed across all browsers? I tried adding something like this at one point and had to remove it for compatibility reasons I can't remember now

Reply:
Okay, let me check. This should be covered by the babel es2015 preset.

Reply:
Yes, these function default parameters should be transformed by the babel parameters transform. Is there a particular browser that you would expect this to not work, if it were not properly transformed?